hawkbit-extended-runtimes/hawkbit-update-server-azure/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM openjdk:8u171-jre-alpine
+
+MAINTAINER Kai Zimmermann <kai.zimmermann@microsoft.com>
+
+ENV HAWKBIT_HOME=/opt/hawkbit
+
+EXPOSE 8080
+
+VOLUME "$HAWKBIT_HOME/data"
+
+WORKDIR $HAWKBIT_HOME
+ENTRYPOINT ["java","-jar","hawkbit-update-server.jar","-Xmx768m -Xmx768m -XX:MaxMetaspaceSize=250m -XX:MetaspaceSize=250m -Xss300K -XX:+UseG1GC -XX:+UseStringDeduplication -XX:+UseCompressedOops -XX:+HeapDumpOnOutOfMemoryError"]
+
+# Add the service itself
+ARG JAR_FILE
+ADD target/hawkbit-update-server-azure-0.3.0-SNAPSHOT.jar hawkbit-update-server.jar
```

Comment:
I won't hardcode the version here. Ah, I see that was the intention for `JAR_FILE` 👍

Reply:
I agree, we should find a better solution for that. However, right now its for manual build only anyhow.

Reply:
Well, maybe you could that a default for the `ARG` and use it within the path. So you could pass a different value at build time